frontend/src/views/web/challenge/participate.html

```diff
@@ -62,7 +62,7 @@
                 <div class="col s12 m6">
                     <div class=" ev-card-panel ev-z-depth-5">
                         <div class="ev-md-container ev-panel-title"><strong>Create a New Team</strong></div>
-                        <div class="loader-container card-loader-container" id="loader" ng-show="challenge.isExistLoader" class="fade">
+                        <div class="loader-container card-loader-container" id="loader" ng-show="challenge.isLoader" class="fade">
```

Comment:
This is the `ng-show` variable where I've changed from `isExistLoader` to `isLoader` to differentiate the showing functionality of left-loader and right-loader.